xbmc/guilib/guiinfo/SystemGUIInfo.cpp

```diff
@@ -570,6 +573,20 @@ bool CSystemGUIInfo::GetBool(bool& value, const CGUIListItem *gitem, int context
     case SYSTEM_SHOW_EXIT_BUTTON:
       value = CServiceBroker::GetSettingsComponent()->GetAdvancedSettings()->m_showExitButton;
       return true;
+    case SYSTEM_IS_HDR_DISPLAY_OFF:
+#if defined(TARGET_WINDOWS)
```

Comment:
please avoid using ifdefs in common sources

Reply:
cannot be easily avoided

Reply:
You may be better off to provide a method via windowing and use cservicebroker to call getwindowingcomponent. Then the platform specific implementation is in platform specific code area

Reply:
ToDo for Team Kodi.....

Reply:
> ToDo for Team Kodi.....

are you kidding?

Reply:
> ToDo for Team Kodi.....

are you kidding?

Reply:
DONE